src/LineChart.js

```diff
@@ -17,32 +17,25 @@ export default class LineChart extends React.Component {
     data: PropTypes.array.isRequired,
     /**
      * Accessor function for line X values, called once per datum, or a single value to be used for the entire line.
-     */
-    x: CustomPropTypes.valueOrAccessor,
+     */ x: CustomPropTypes.valueOrAccessor,
     /**
      * Accessor function for line Y values, called once per datum, or a single value to be used for the entire line.
-     */
-    y: CustomPropTypes.valueOrAccessor,
+     */ y: CustomPropTypes.valueOrAccessor,
```

Comment:
Is your editor removing this space for some reason?

Reply:
I'm using vscode which does prettier on save. Lemme deal with that